test/README.md

```diff
@@ -1,17 +1,19 @@
 ## How to execute tests
 
+It requires [vim-themis](https://github.com/thinca/vim-themis).
+
 ```console
-$ bundle install --path=.bundle
-$ rake test
+$ git clone https://github.com/thinca/vim-themis
+$ ./vim-themis/bin/themis .
 ```
 
-## How to watch file changes and run tests automatically
+## How to measure code coverage
+
+It requires [covimerage](https://github.com/Vimjas/covimerage).
 
 ```console
-$ rake guard
+$ pip install covimerage
+$ PROFILE_LOG=profile.txt ../vim-themis/bin/themis .
```

Comment:
Please add `$ cd /path/to/clever-f.vim/test` line before this line to clarify where the current working directory is.